packages/devtools_app/benchmark/devtools_benchmarks_test.dart

```diff
@@ -173,50 +173,62 @@ void _verifyScoresAgainstThresholds(
 const _frameTimeFor60FPSInMicros = 16666.6;
 
 final _benchmarkThresholds = {
+  // Note that some of these benchmarks exceed the 60fps frame budget,
+  // especially the p90 benchmarks.
+  //
+  // See https://github.com/flutter/devtools/pull/8892 which exposes this.
   DevToolsBenchmark.navigateThroughOfflineScreens: {
     ..._valuesForMetric(
       BenchmarkMetric.flutterFrameTotalTime,
-      avg: _frameTimeFor60FPSInMicros,
+      avg: _frameTimeFor60FPSInMicros * 2,
       p50: _frameTimeFor60FPSInMicros,
-      p90: _frameTimeFor60FPSInMicros,
+      p90: _frameTimeFor60FPSInMicros * 6,
     ),
     ..._valuesForMetric(
       BenchmarkMetric.flutterFrameBuildTime,
-      avg: _frameTimeFor60FPSInMicros,
+      avg: _frameTimeFor60FPSInMicros * 2,
       p50: _frameTimeFor60FPSInMicros,
       p90: _frameTimeFor60FPSInMicros,
     ),
     ..._valuesForMetric(
       BenchmarkMetric.flutterFrameRasterTime,
-      avg: _frameTimeFor60FPSInMicros,
+      avg: _frameTimeFor60FPSInMicros * 2,
       p50: _frameTimeFor60FPSInMicros,
       p90: _frameTimeFor60FPSInMicros,
     ),
   },
+  // Note that some of these benchmarks exceed the 60fps frame budget,
+  // especially the p90 benchmarks.
+  //
+  // See https://github.com/flutter/devtools/pull/8892 which exposes this.
   DevToolsBenchmark.offlineCpuProfilerScreen: {
     ..._valuesForMetric(
       BenchmarkMetric.flutterFrameTotalTime,
-      avg: _frameTimeFor60FPSInMicros,
+      avg: _frameTimeFor60FPSInMicros * 2,
       p50: _frameTimeFor60FPSInMicros,
-      p90: _frameTimeFor60FPSInMicros,
+      p90: _frameTimeFor60FPSInMicros * 6,
     ),
     ..._valuesForMetric(
       BenchmarkMetric.flutterFrameBuildTime,
-      avg: _frameTimeFor60FPSInMicros,
+      avg: _frameTimeFor60FPSInMicros * 2,
       p50: _frameTimeFor60FPSInMicros,
       p90: _frameTimeFor60FPSInMicros,
     ),
     ..._valuesForMetric(
       BenchmarkMetric.flutterFrameRasterTime,
-      avg: _frameTimeFor60FPSInMicros,
+      avg: _frameTimeFor60FPSInMicros * 2,
       p50: _frameTimeFor60FPSInMicros,
       p90: _frameTimeFor60FPSInMicros,
     ),
   },
+  // Note that some of these benchmarks exceed the 60fps frame budget,
+  // especially the p90 benchmarks.
+  //
+  // See https://github.com/flutter/devtools/pull/8892 which exposes this.
   DevToolsBenchmark.offlinePerformanceScreen: {
     ..._valuesForMetric(
       BenchmarkMetric.flutterFrameTotalTime,
-      avg: _frameTimeFor60FPSInMicros,
+      avg: _frameTimeFor60FPSInMicros * 1.5,
```

Comment:
do we need to increase the threshold on this one? The code in this PR should not have affected this benchmark at all.

Reply:
It could have been a flake, but I did see these fail.

I don't know what all this is actually measuring, but if it involved the loading of a profile then it could have been affected.

Reply:
Oh I see this is actually a totally different screen. I will try reverting this one and see if it passes now.